python-frontend/src/main/java/org/sonar/python/cfg/fixpoint/UsageVisitor.java

```diff
@@ -70,6 +73,7 @@ private void addToSymbolToUsageMap(Usage usage, Symbol symbol) {
   public static final class SymbolUsage {
```

Comment:
@andrea-guarino-sonarsource This name is confusing to me. What's the difference between `Usage` and `SymbolUsage`? What about renaming it to something like `SymbolReadWrite`? we could also think about renaming `UsageVisitor`, `getVariableUsages`...

Reply:
Yes, good point. I agree with renaming it to `SymbolReadWrite` to avoid confusion.